[2024-09-22T01:32:12.360+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:32:04.704794+00:00 [queued]>
[2024-09-22T01:32:12.380+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:32:04.704794+00:00 [queued]>
[2024-09-22T01:32:12.380+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-09-22T01:32:12.406+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_cross_validation.RandomForestRegressor> on 2024-09-22 01:32:04.704794+00:00
[2024-09-22T01:32:12.420+0000] {standard_task_runner.py:60} INFO - Started process 6114 to run task
[2024-09-22T01:32:12.429+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_eval', 'model_cross_validation.RandomForestRegressor', 'manual__2024-09-22T01:32:04.704794+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpl76nmgff']
[2024-09-22T01:32:12.431+0000] {standard_task_runner.py:88} INFO - Job 168: Subtask model_cross_validation.RandomForestRegressor
[2024-09-22T01:32:12.454+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-09-22T01:32:12.525+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:32:04.704794+00:00 [running]> on host cc45b92b61e2
[2024-09-22T01:32:12.787+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_eval' AIRFLOW_CTX_TASK_ID='model_cross_validation.RandomForestRegressor' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T01:32:04.704794+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T01:32:04.704794+00:00'
[2024-09-22T01:32:13.540+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-09-22T01:32:13.554+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_eval, task_id=model_cross_validation.RandomForestRegressor, execution_date=20240922T013204, start_date=20240922T013212, end_date=20240922T013213
[2024-09-22T01:32:13.617+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-09-22T01:32:13.652+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
